frontend/src/Components/Dashboards/DashboardVolunteers.jsx

```diff
@@ -9,70 +9,80 @@ DashboardVolunteers Component | Capstone App (Pursuit Volunteer Mgr)
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
-import EventPreviewCard from '../EventPreviewCard';
+import EventsDashVolunteers from './EventsDash/EventsDashVolunteers';
 import EventCard from '../EventCard';
+// import EventPreviewCard from '../EventPreviewCard';
 
 const Dashboard = (props) => {
     const { setFeedback, loggedUser } = props;
 
-    const [eventsList, setEventsList] = useState([]);
+    const [eventsObj, setEventsObj] = useState({ upcomings: [], pasts: [], importants: [] });
     const [showEvent, setShowEvent] = useState(false);
-    const [volunteeredTime, setVolunteeredTime] = useState(0);
-    const [pastEvents, setPastEvents] = useState(0);
-    const [importantEvents, setImportantEvents] = useState([]);
     const [targetEvent, setTargetEvent] = useState({});
+    const [volunteeredTime, setVolunteeredTime] = useState(0);
     const [reloadDashboard, setReloadDashboard] = useState(false);
+    // const [eventsList, setEventsList] = useState([]);
+    // const [pastEvents, setPastEvents] = useState(0);
 
 
     useEffect(() => {
-        const getEvents = async () => {
-            try {
-                const { data } = await axios.get(`/api/events/upcoming/volunteer/${props.loggedUser.v_id}?limit=3`);
-                setEventsList(data.payload);
-            } catch (err) {
-                setFeedback(err)
-            }
-        }
-
-        const getImportantEvents = async () => {
-            try {
-                const { data } = await axios.get(`/api/events/important?limit=3`);
-                setImportantEvents(data.payload);
-
-            } catch (err) {
-                setFeedback(err)
-            }
+        // const getEvents = async () => {
+        //     try {
+        //         const { data } = await axios.get(`/api/events/upcoming/volunteer/${props.loggedUser.v_id}?limit=3`);
+        //         setEventsList(data.payload);
+        //     } catch (err) {
+        //         setFeedback(err)
+        //     }
+        // }
+        // const getImportantEvents = async () => {
+        //     try {
+        //         const { data } = await axios.get(`/api/events/important?limit=3`);
+        //         setImportantEvents(data.payload);
+        //     } catch (err) {
+        //         setFeedback(err)
+        //     }
+        // }
+        // getEvents();
+        // getImportantEvents();
+
+        const getEventsData = () => {
+          axios.get(`/api/events/non_admin/dashboard`)
+            .then(res => setEventsObj(res.data.payload))
+            .catch(err => setFeedback(err));
         }
-
-        getEvents();
-        getImportantEvents();
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [reloadDashboard]);
-
-    useEffect(() => {
         const getAllVolunteeredTime = async () => {
```

Comment:
No need to async here
Also it's more efficient if we do this network request in it's own useEffect that has an empty array of dependencies. No need to do a network request to have the volunteered hours when a volunteer signs up to volunteer for an event or remove their request to volunteer